### ISSUE TITLE
A11y_VscodeWebviewUiToolkit_AllComponents_Panels_ScreenReader: NVDA does not announces the name of the selected tab items like "Problems" when navigating using tab key

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/vscode-webview-ui-toolkit#186]
### Actual:
NVDA does not announces the name of the selected tab items like "Problems" when navigating using tab key.

Similar issue is observed with Jaws

Similar issue is with all the Panels and Panel's tab item

### Expected:
NVDA should be announcing the name of the selected tab items like "Problems" when navigating using tab key

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_VscodeWebviewUiToolkit_Aug2021;#Webview UI Toolkit for Visual Studio Code;#Win32;#FTP;#DesktopApp;#WCAG1.3.1;#NVDA;#Jaws;#STP;#AINotInScope;

### Environment Details:
Application Name: Vscode Webview Ui Toolkit
Windows Version: Win10
Screen Reader: NVDA(2021.1), Jaws(2021.2102.34)

### Repro Steps:

1. Launch VS Code
2. Open WebView UI Toolkit: All Component extension
3. Tab to Panels and tab to "Problems" tab item
4. Start NVDA use tab key and check the announcement

### User Impact:
Screen Reader users will not get the proper information about the tab items in the panels

[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Comment by fsteffi** [comment]
**GithubTags:**#Closed;
